### PR TITLE
Improving lineage reconstruction

### PR DESCRIPTION
**Pull request by ggabernet**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
  - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/airrflow/tree/master/.github/CONTRIBUTING.md)
  - [ ] If necessary, also make a PR on the nf-core/airrflow _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: nextflow_schema.json [hunk 1]
```diff
@@ -168,7 +168,7 @@
             "fa_icon": "fas fa-barcode"
         },
         "presto_options": {
-            "title": "Presto options",
+            "title": "pResto options",
```
Comment (ssnn-airr) [inline nextflow_schema.json:171]: If you want to maintain Immcantation's style: pRESTO (https://presto.readthedocs.io/en/stable/)

File: nextflow_schema.json [hunk 2]
```diff
@@ -272,6 +272,29 @@
...
         },
+        "report_options": {
+            "title": "Report options",
+            "type": "object",
+            "description": "",
+            "default": "",
+            "properties": {
+                "report_rmd": {
+                    "type": "string",
+                    "default": "/Users/gisela/Projects/Pipelines/nf-core-bcellmagic/assets/repertoire_comparison.Rmd",
```
Comment (ssnn-airr) [inline nextflow_schema.json:283]: Is the "/Users/gisela/Projects/Pipelines/nf-core-bcellmagic" part of the path correct?  Similar below, in report_css and report_logo.

**Review by ssnn-airr — APPROVED** [review]

File: nextflow_schema.json (hunk 2 above)
Comment (ggabernet) [reply nextflow_schema.json]: Ah, thanks for the catch, I've updated it now

File: nextflow_schema.json (hunk 1 above)
Comment (ggabernet) [reply nextflow_schema.json]: right, thanks!

**Comment by ggabernet** [comment]
merging without all tests passing as nf-core linting will be fixed in next PR

**Pull request merged**